app/services/container_dashboard_service.rb

```diff
@@ -49,6 +51,14 @@ def image_metrics_data
     }.compact
   end
 
+  def refresh_status_data
+    {
+      :defaultCred     => ( @ems.present? && @ems.enabled? ) ? 'Valid': 'Invalid',
+      :lastRefreshDate => @ems.present? ? refresh_date(@ems) : [],
+      :lastRefresh     => @ems.present? ? refresh_status(@ems) : []
```

Comment:
Ruby is usually in snake_case, so this should be `last_refresh` (and similar changes elsewhere in the ruby code (unless there was an interop reason with JS)

Reply:
@Fryguy means, lastRefreshDate is last_refersh_date?, it seems to be working fine.